board/embeddedartists/mx7ulpea-ucom/mx7ulpea-ucom.c

```diff
@@ -4,7 +4,6 @@
  *
  * SPDX-License-Identifier:	GPL-2.0+
  */
-
```

Comment:
nit: can you not remove this line? Just to avoid possible conflicts later on as we rebase the patches.

Reply:
sure, let me repost